### ISSUE TITLE
[Bug] Player with characters that have old currency, cannot open LootSheets

### ISSUE THREAD

**Issue by Dairid** [jopeek/fvtt-loot-sheet-npc-5e#294]
Certain players (most of them) cannot open any loot sheets regardless of permissions.  One player can, the rest cannot, there are no configuration differences that I can see on their ends.  I've updated to the newest version but it still persists.  Owner/Observer has the same behavior (as in no sheet will open for them).  If I swap them to a different sheet type it will open, but lootsheet will not.

**Comment by Dairid** [comment]
I was able to resolve this issue with some fenagling.

Basically it seems like there was some sort of token or something that was associated with my characters (most likely from 8.x).  I use a player importer tool and had not been syncing currency for my existing players for quite some time, the one player who it was working for was new and I had freshly synced his currency as he was a new player.  I went and did a manual import of my players currency using my importer tool and it cleared up the issue.
